nixos/tests/postgresql.nix

```diff
@@ -69,5 +69,47 @@ let
 in
   (mapAttrs' (name: package: { inherit name; value=make-postgresql-test name package false;}) postgresql-versions) // {
     postgresql_11-backup-all = make-postgresql-test "postgresql_11-backup-all" postgresql-versions.postgresql_11 true;
+
+    postgresql_dirmode_change =
```

Comment:
+1 for updating tests

Reply:
I had to remove it, as I don't want to support old pgs, and new ones work fine.